versioned_docs/version-2.0.0/quickstart/go-gin-redis.md

```diff
@@ -0,0 +1,638 @@
+---
+id: samples-gin
+title: Sample User Authentication App (Golang)
+sidebar_label: Golang - Gin + Redis
+description: The following sample app showcases how to use Gin framework and the Keploy Platform.
+tags:
+  - Gin Framework
+  - Redis
+keyword:
+  - Gin Framework
+  - Redis
+  - Golang
+  - API Test generator
+  - Auto Testcase generation
+---
+
+## Introduction
+
+🪄 Dive into the world of User Authentication apps and see how seamlessly Keploy integrates with [Gin](https://gin-gonic.com/) and [Redis](https://www.redis.io/). Buckle up, it's gonna be a fun ride! 🎢
+
+## Pre-Requisite 🛠️
+
+- Install WSL (`wsl --install`) for <img src="/docs/img/os/windows.png" alt="Windows" width="3%" /> Windows.
+- Install Colima( `brew install colima && colima start` ) for <img src="/docs/img/os/macos.png" alt="MacOS" width="3%" /> MacOs.
+
+## Get Started! 🎬
+
+## Setup the Redis Database 📦
+
+  Create a docker network, run -
+
+  ```bash
+  docker network create <network-Name>
+  ```
+
+  Start the Redis instance using the `docker-compose` file-
+
+  ```bash
+  sudo docker run -p 6379:6379 -d --network <networkName> --name myredis redis
+  ```
+
+  ```bash
+  docker build -t gin-app:1.0 .
+  ```
+
```

Comment:
I think this section will be after the cloning the repository and cd gin-redis

Reply:
Done!!